run_fvs.sh

```diff
@@ -12,13 +12,23 @@ git status
 docker exec -t host1 bash -c 'docker rm -f $(docker ps -qa) ; \
                               docker rmi $(docker images -qa)' || true
 docker rm -f host1 || true
+
+docker pull busybox:latest
```

Comment:
While I was watching this run, it does actually sit at `docker pull` for several seconds, even if it eventually concludes that it doesn't need to pull the image. I used the `docker images | grep || docker pull` on the etcd one because it was pinned at a version, so presumably we wouldn't care about getting the latest. But if we don't care, I can use the `if [` on all of them.